grype/search/distro.go

```diff
@@ -17,6 +18,11 @@ func ByPackageDistro(store vulnerability.ProviderByDistro, d *distro.Distro, p p
 		return nil, nil
 	}
 
+	if isUnknownVersion(p.Version) {
+		log.WithFields("package", p).Warn("skipping package with unknown version")
```

Comment:
Does this want to be `.WithFields("package", p.Name)`? Sometimes logging a whole struct doesn't look great. What does it look like when we log the package?